src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -88,7 +87,12 @@ class CLIJsonWebKeyOperation(str, Enum):
         c.argument('sku', arg_type=get_enum_type(SkuName, default=SkuName.standard.value))
         c.argument('no_self_perms', arg_type=get_three_state_flag(), help="Don't add permissions for the current user/service principal in the new vault.")
         c.argument('location', validator=get_default_location_from_resource_group)
-        c.argument('enable_soft_delete', arg_type=get_three_state_flag(), help='Enable vault deletion recovery for the vault, and all contained entities. If omitted, assume true as default value.')
+        c.argument('enable_soft_delete', arg_type=get_three_state_flag(), help="Enable 'soft delete' functionality for this key vault and all contained entities. If omitted, it will be set to true by default. Once set to true, it cannot be reverted to false.")
```

Comment:
Instead of previously assigning `enable_soft_delete` to both `create` and `update` under `keyvault`, and then override the one in `create`, I separated `create` and `update` so that there is no overriding relationship for `enable_soft_delete` and `retention_days` .